archinstall/lib/installer.py

```diff
@@ -1,9 +1,9 @@
 from .disk import *
 from .hardware import *
-from .locale_helpers import verify_keyboard_layout, verify_x11_keyboard_layout
+from .locale_helpers import verify_x11_keyboard_layout
```

Comment:
Can we please stop removing imports that are being used?

Reply:
There's clearly something wrong with PyCharm if it's removing used imports when I tell it to optimize.

Trying to figure this out still